corehq/apps/app_manager/util.py

```diff
@@ -27,6 +27,8 @@
     USERCASE_ID,
     USERCASE_PREFIX,
     USERCASE_TYPE,
+    MOBILE_UCR_V1_FIXTURE_IDENTIFIER,
+    MOBILE_UCR_V1_ALL_REFERENCES,
```

Comment:
See https://github.com/dimagi/commcare-hq/pull/35168/files#r1800689494 for the reason

```suggestion
    MOBILE_UCR_V1_ALL_REFERENCES,
    V1_CASE_LIST_REFERENCES_PATTERN,
```

Reply:
Addressed in [0cf6206](https://github.com/dimagi/commcare-hq/pull/35168/commits/0cf6206de2829c5b157acdfaf062aef9fd662fdf)